app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -179,7 +176,10 @@ public void run() {
         logger.info(String.format("Analyzing %s", analysisName));
         AppResult result = _analyticJob.getAnalysis();
         result.save();
-        logger.info(String.format("Analysis of %s took %sms", analysisName, System.currentTimeMillis() - analysisStartTimeMillis));
+        long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
+        logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
+        MetricsController.setJobProcessingTime(processingTime);
```

Comment:
We could do that. However with the fast fetcher this time is pretty small (milliseconds) and this metric is mostly to alert if it goes way up. It takes a long time with the regular fetcher due to each task requiring a separate HTTP request.
@shankar37 If you want I'll add the normalized version too. I have an idea for how to do it cleanly.